content/blog/fix-iam-trust-policy-errors-with-aws-sso-regional-arn.md

```diff
@@ -0,0 +1,188 @@
+---
+visible: true
+draft: false
+title: "Fixing IAM Trust Policy Errors when using AWS IAM Identity Center SSO Role (Regional ARN, us-east-1 vs other regions)"
```

Comment:
I'd shorten this, something like "Fix IAM Trust Policy Errors with AWS IAM Identity Center SSO Role Regional ARN". Still should be good for SEO.